src/server/ApiRoutes/User.js

```diff
@@ -6,7 +6,11 @@ function requireUser(req, res) {
     !req.patronTokenResponse.decodedPatron || !req.patronTokenResponse.decodedPatron.sub) {
     // redirect to login
     const fullUrl = encodeURIComponent(`${req.protocol}://${req.get('host')}${req.originalUrl}`);
-    res.redirect(`${appConfig.loginUrl}?redirect_uri=${fullUrl}`);
+    if (fullUrl.includes('api')) {
```

Comment:
Could this be a little more selective - like `/api/`?

Reply:
Included `%2Fapi%2F` as the check